frontend/packages/schema-editor/src/components/SchemaInspector/ItemDataComponent.tsx

```diff
@@ -153,7 +155,7 @@ export function ItemDataComponent({ language, selectedItem }: IItemDataComponent
       {selectedItem.objectKind === ObjectKind.Field && (
         <Select
           label={t('type')}
-          onChange={(type: string) => onChangeFieldType(selectedItem.pointer, type as FieldType)}
+          onChange={(type: FieldType) => onChangeFieldType(type)}
```

Comment:
Fungerer dette knirkefritt? Mener å huske at det forårsaket noen problemer å gjøre det på denne måten siden den venter å få inn en string, men det var kanskje noe annet. Det burde jo egentlig ikke være et problem.

Reply:
Fikk ingen feil på det i min IDE og heller ikke ved compile. Tenkte på det samme som du skriver, men siden jeg ikke fikk noen feil, så ser det greit ut. ☺️